hosts/signed-int16/home/reisen/programs/discord.nix

```diff
@@ -0,0 +1,16 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  discordOverlay = pkgs.discord.override {
+    withOpenASAR = true;
+    withVencord = true;
+    # withTTS = true;
+  };
+in {
+  home.packages = [
+    discordOverlay
+  ];
+}
```

Comment:
this file shouldn't exist at all, because we already have a package at `modules/shared/home/overlay`